pom.xml

```diff
@@ -63,7 +63,19 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-compress</artifactId>
-        <version>1.26.1</version>
+        <version>1.26.2</version>
+        <!-- [CVE-2024-47554] in commons-io:commons-io:jar:2.11.0 -->
+        <exclusions>
+          <exclusion>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
+      <dependency>
+        <groupId>commons-io</groupId>
+        <artifactId>commons-io</artifactId>
+        <version>2.18.0</version>
```

Comment:
It could be nice to use a property for the commons-io version.

Reply:
Done with properties for version of commons-compress, commons-io and h2